### PR TITLE
MB-12542 Move and shipment info for evaluation report forms

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Summary

This adds the ShipmentDisplay component to the ShipmentEvaluationReport page. This also meant needing to add handlers for getting a single evaluation report, a single MTOShipment, and making sure the services infrastructure supported those things effectively. There will be another PR coming to cover adding this information to a counseling evaluation report screen.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the office app.
2. Login as a QAE/CSR.
3. Find a move, if you're looking locally or at an ephemeral deploy you can search for `SITEXT` (move code).
5. Click on the quality assurance tab.
6. Click to create or view a report.
7. Revel in the shipment/move info you see.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

## Screenshots
image

### CODE REVIEW

File: pkg/handlers/ghcapi/api.go [hunk 1]
```diff
@@ -81,14 +81,24 @@ func NewGhcAPIHandler(handlerConfig handlers.HandlerConfig) *ghcops.MymoveAPI {
 		CustomerSupportRemarkDeleter: customerserviceremarks.NewCustomerSupportRemarkDeleter(),
 	}
 
+	ghcAPI.EvaluationReportsCreateEvaluationReportForShipmentHandler = CreateEvaluationReportHandler{
+		HandlerConfig:           handlerConfig,
+		EvaluationReportCreator: evaluationreport.NewEvaluationReportCreator(),
+	}
```
Comment (PDickeyTruss) [inline pkg/handlers/ghcapi/api.go:87]: I think this one got duplicated, I think I see it on line 104 as well (probably due to merging madness)

Comment (Ryan-Koch) [reply pkg/handlers/ghcapi/api.go:87]: Good catch! Just pushed a commit to remove the duplicate.

File: pkg/handlers/ghcapi/mto_shipment.go [hunk 2]
```diff
@@ -52,14 +53,6 @@ func (h ListMTOShipmentsHandler) Handle(params mtoshipmentops.ListMTOShipmentsPa
...
 			}
 
-			if !appCtx.Session().IsOfficeUser() ||
-				(!appCtx.Session().Roles.HasRole(roles.RoleTypeServicesCounselor) &&
-					!appCtx.Session().Roles.HasRole(roles.RoleTypeTOO) &&
-					!appCtx.Session().Roles.HasRole(roles.RoleTypeTIO) &&
-					!appCtx.Session().Roles.HasRole(roles.RoleTypeQaeCsr)) {
-				return handleError(apperror.NewForbiddenError("user is not an office user or does not have a SC, TOO, or TIO role"))
-			}
-
```
Comment (PDickeyTruss) [inline pkg/handlers/ghcapi/mto_shipment.go:62]: Love to see this go 🎉

**Review by PDickeyTruss — APPROVED** [review]
LGTM, excellent building blocks in this PR for us to continue building out reports on 💯

**Review by cosburn — APPROVED** [review]
Looks good! Approving with the note that a future PR will cover design for the shipment card component.

**Pull request merged**